packages/api/src/env.ts

```diff
@@ -9,6 +9,8 @@ export const env = setEnv({
     nodeEnv: 'NODE_ENV',
     port: 'PORT',
     databaseUrl: 'DATABASE_URL',
+    GitHubId: 'GITHUB_CLIENT_ID',
```

Comment:
We want to keep the naming of our variables consistent with what they're storing.
```suggestion
    GitHubClientId: 'GITHUB_CLIENT_ID',
```